packages/chip/src/Chip.stories.tsx

```diff
@@ -41,7 +45,7 @@ const meta: StoryMetaType<typeof Chip> = {
   args: {
```

Comment:
can we update the `glyph` control to match the select control we use for `leftGlyph`/`rightGlyph` in the button stories? https://github.com/mongodb/leafygreen-ui/blob/c86227a67d34bae8b5c858b2a72bb348fa1332df/packages/button/src/Button.stories.tsx#L56-L74